lib/modules/manager/helm-values/extract.ts

```diff
@@ -46,7 +46,8 @@ function findDependencies(
       let registry = currentItem.registry;
       registry = registry ? `${registry}/` : '';
       const repository = String(currentItem.repository);
-      const tag = String(currentItem.tag);
+      let tag = currentItem.tag;
+      tag = tag ? String(tag) : String(currentItem.version);
```

Comment:
```suggestion
      const tag = `${currentItem.tag ?? currentItem.version}`;
```
simplify

Reply:
We should validate, that the yaml parser extracted that as strings and log a warning if it's a number.

If  `tag: 1.2.10` then the yaml parser will read `1.2.1` as number (yaml standard).

Can be done in seperate PR

Reply:
`tag` is the same as before, what is added here is the ability to parse the version from a `version` field

Reply:
please apply my suggestion

Reply:
done